### PR TITLE
M2C and C2M crons only run on correct Saturdays

### PR DESCRIPTION
**Pull request by ndushay**
It turns out this cron:
```
0 5 1-7 * 6
```
runs each day numbered between 1-7 as well as on every Saturday.   So I took a kludge from the interwebs to fix it,  so we *only* run M2C and C2M on the appropriate Saturdays.

I also tweaked the README:
- added a linked TOC
- made the Headings more skimmable
- moved the seeding section down

Connects to #810

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -1,22 +1,24 @@
 # Use this file to easily define all of your cron jobs.
 # Learn more: http://github.com/javan/whenever
 
-# run these task only on servers with the correct roles in Capistrano
-# these tasks append to existing logs
-every '0 5 1-7 * 6', roles: [:m2c] do
+# also overriding b/c whenever adds --silent (https://github.com/javan/whenever/issues/524, #541)
+job_type :sat_only_rake, "cd :path && bin/is_it_saturday.sh && :environment_variable=:environment :bundle_command rake :task :output"
```
Comment (atz) [inline config/schedule.rb]: I don't know if it is an issue here (can't see the file mode in github view), but `bash bin/is_it_saturday.sh` will succeed when the script is just readable, whereas direct invocation requires it to be executable.

Comment (atz) [reply config/schedule.rb]: Ah, OK, if I go view the file itself, GitHub shows it as an `Executable File`, so no problem.

File: README.md [hunk 2]
```diff
@@ -206,7 +172,66 @@ this will generate a log at, for example, `log/profile_cv_validate_disk_all_endp
...
+RAILS_ENV=production bundle exec rake drop[fixture_sr1]
+```
+
+#### Populate the catalog:
+
+```sh
+RAILS_ENV=production bundle exec rake populate[fixture_sr1]
+```
+
+## <a name="development"/>Development
```
Comment (atz) [inline README.md]: Headings automatically have anchors added, so when the anchor isn't changing, it is redundant to add explicitly.  Github already renders the markdown with an anchor, e.g. for `### Redis`, like:
```html
<h3><a id="user-content-redis" class="anchor" aria-hidden="true" href="#redis"></a>Redis</h3>
```

Comment (ndushay) [reply README.md]: ah - I wasn't sure, and couldn't think of a good way to test without pushing to github.

Comment (jmartin-sul) [reply README.md]: merged, filed #832 for this loose end.

**Pull request merged**